Geometry/HGCalCommonData/data/hgcalMaterial/v1/hgcalMaterial.xml

```diff
@@ -10,7 +10,7 @@
       <rMaterial name="materials:Copper"/>
     </MaterialFraction>
   </CompositeMaterial>
-  <CompositeMaterial name="HGC_G10-FR4" density="1.85*g/cm3" symbol=" " method="mixture by weight">
+  <CompositeMaterial name="HGC_G10-FR4" density="1.70*g/cm3" symbol=" " method="mixture by weight">
```

Comment:
@bsunanda - shouldn't be a v2 of hgcalMaterial.xml?

Reply:
Hi Yana - D28 is not yet in production yet. No relval is made officially for this. That is why I do not want to make it v2. Otherwise I have to change Configuration/Geometry and Geometry/CMSCommonData. I can do that if you think worthwhile